dali/sasha/sasha.cpp

```diff
@@ -374,23 +375,11 @@ ISashaCommand *createCommand(unsigned argc, char* argv[], SocketEndpoint &server
     return cmd.getClear();
 }
 
-bool getVersion(INode *node)
+void getVersion(INode *node)
```

Comment:
see other comments,
It would be better if each command implementation was contained in saruncmd.

The interaction with a directly accessible sasha 'server' should continue to work as it does now for the time being.
But there should also be a way to interact with the new esp service via the CLI, e.g. by specifying "service=<esp-service-endpoint>" instead of "server=<sasha-server>"

The implementation of this CLI to the service will necessarily have to look quite different, using the new saruncmd's.
It's a good reason to add just a few command for now, in order to get the structure right.
Ultimately, "createCommand" may need refactoring, since it's doing serveral things:
1) parsing the command line
2) creating an ISashaCommand as a repository for the parsed command (which also implements server<>client send/receipt)
3) interactive behaviour (via 'confirm').

Continuing to use it to parse the command line into a ISashaCommand is probably fine/pragmatic, which can then be used to decide what/how to run selected saruncmd's after that when talking to the esp service.